R/format_ae_specific.R

```diff
@@ -43,12 +43,16 @@ format_ae_specific <- function(outdata,
                                digits_events = c(1, 1),
                                mock = FALSE) {
   display <- tolower(display)
-
   display <- match.arg(display,
-    c("n", "prop", "total", "diff", "diff_ci", "p", "dur", "mean", "events"),
+    c("n", "prop", "total", "diff", "diff_ci", "diff_p", "dur", "events"),
     several.ok = TRUE
   )
 
+  # Add "n"
+  if(!"n" %in% display){
```

Comment:
How about 

`display <- unique(c("n", display))` ?

Reply:
Agree! See updated PR below.

Reply:
After this change, you do not need the if statement.